.github/workflows/my-ci-pipline.yml

```diff
@@ -16,4 +16,8 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - run: docker build --target test --tag todo-app:test . 
-    - run: docker run todo-app:test   
\ No newline at end of file
+    - run: docker run todo-app:test   
+
+#I have not dont the module 8 yet. I need to use the curl command in future 
+ #final step 
+#curl -dH -X POST ${{secrets.WEBHOOK_URL}}
```

Comment:
Note that I don't think you've made the changes to your Azure App Service directly either, so we'll want to do that once the CD is set up (or even just make them directly when merging this!)